ods_ci/tests/Resources/Page/Operators/OpenShiftPipelines.resource

```diff
@@ -0,0 +1,20 @@
+*** Settings ***
+Documentation       Test suite for OpenShift Pipeline
+Resource            ../../RHOSi.resource
+Resource            ../../ODS.robot
+Resource            ../../Common.robot
+Resource            ../../Page/ODH/ODHDashboard/ODHDashboard.robot
+Library             DateTime
+Library             ../../libs/DataSciencePipelinesAPI.py
+
+
+*** Variables ***
+${REDHAT_OPENSHIFT_PIPELINES_YAML}=            ods_ci/tests/Resources/Files/redhat-openshift-pipelines.yaml
+
+
+*** Keywords ***
+Install Red Hat OpenShift Pipelines
+    [Documentation]    Install Red Hat OpenShift Pipelines
+    Oc Apply    kind=Subscription    src=${REDHAT_OPENSHIFT_PIPELINES_YAML}
+    ${pod_count}    Wait Until OpenShift Pipelines Operator Is Deployed
+    Should Be True    ${pod_count} == 1    All the pods were created
```

Comment:
is this file still required? I thought we replaced this in the PR #767 by @tarukumar

Reply:
Diego moved Tarun's code here so it can be used in different test suites